storage/docs/commit_log.md

```diff
@@ -0,0 +1,4 @@
+# Commit Log
```

Comment:
@mambisi 
please remove this commit_log.md file and add short descrption to `storage/README.md`